contractcourt/taproot_briefcase.go

```diff
@@ -71,6 +78,7 @@ func (t *taprootBriefcase) Encode(w io.Writer) error {
 
 // Decode decodes the given reader into the target struct.
 func (t *taprootBriefcase) Decode(r io.Reader) error {
+	commitBlob := t.CommitBlob.Zero()
```

Comment:
Why not add this to the `DecodeRecords`?

Reply:
Good q, the way things work is that we need to have a ref to the original TLV record, so we can set it later after the call to `DecodeWithParsedTypes`. The `Zero()` method actually creates a _new_ record, which isn't bound to the original struct like the rest of the items in `DecodeRecords` are.